### PR TITLE
Hide main window when dragging results

### PR DESCRIPTION
**Pull request by VictoriousRaptor**
Hide main window when dragging results, so users can see where they're dragging to.

### CODE REVIEW

File: Flow.Launcher/ResultListBox.xaml.cs [hunk 1]
```diff
@@ -137,6 +137,8 @@ private void ResultList_MouseMove(object sender, MouseEventArgs e)
 
             isDragging = false;
 
+            App.API.HideMainWindow();
+
```
Comment (VictoriousRaptor) [inline Flow.Launcher/ResultListBox.xaml.cs:141]: Main window will show later when finishing dragging since we're calling `ChangeQuery()` in line 156.

**Review by onesounds — APPROVED** [review]

File: Flow.Launcher/ResultListBox.xaml.cs (hunk 1 above)
Comment (VictoriousRaptor) [reply Flow.Launcher/ResultListBox.xaml.cs:141]: Main window will only show later when it's a Move action now. Should we amend this? @onesounds @jjw24

Comment (jjw24) [reply Flow.Launcher/ResultListBox.xaml.cs:141]: Sorry what do you mean? Does change query call make the search window visible again? When I was testing this with moving a file the search window wasn't showing after the drop was done and had to retrigger window to appear.

Comment (VictoriousRaptor) [reply Flow.Launcher/ResultListBox.xaml.cs:141]: When I was testing this PR something did bring back the search window, and I thought it was ` ChangeQuery`. But now I am confused about the logic. Will test later this week.

**Pull request merged**